apps/bare-expo/android/app/src/main/res/values/styles.xml

```diff
@@ -11,7 +12,9 @@
     <item name="android:textColorHint">#c8c8c8</item>
     <item name="android:textColor">@android:color/black</item>
   </style>
-  <style name="AppTheme.SplashScreen" parent="AppTheme">
-    <item name="android:windowBackground">@drawable/splashscreen</item>
+  <style name="Theme.Expo.SplashScreen" parent="Theme.SplashScreen">
+    <item name="windowSplashScreenBackground">@color/splashscreen_background</item>
+    <item name="windowSplashScreenAnimatedIcon">@drawable/splashscreen_image</item>
```

Comment:
nit: If you plan to add support for brand image later, I suggest to rename `splashscreen_image` -> `splashscreen_logo` (+do the same in the app.json splash config)